__tests__/unit/core-debugger-cli/__fixtures__/block.json

```diff
@@ -1,10 +1,9 @@
 {
```

Comment:
The debugger CLI has been merged into the tester CLI with https://github.com/ArkEcosystem/core/commit/bdd07d3f0d4f9e75d0d8c02827eaf0a088581908.

Reply:
Merged 2.3 -> blockid in 71a87300.